sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java

```diff
@@ -825,6 +825,18 @@ public void testUpdateToRoutedRequestByPlannerWhereOr() throws Exception {
         assertThat(response.rowCount(), is(2L));
     }
 
+    @Test
+    public void testExceptionOnGroupingByArrayType() throws Exception {
+        execute("create table test (array_col array(string))");
```

Comment:
can you do this with an analyzer unittest instead of a full blown integration test?